src/Renci.SshNet/Abstractions/ThreadAbstraction.cs

```diff
@@ -10,24 +10,17 @@ internal static class ThreadAbstraction
         /// <param name="millisecondsTimeout">The number of milliseconds for which the thread is suspended.</param>
         public static void Sleep(int millisecondsTimeout)
         {
-#if FEATURE_THREAD_SLEEP
```

Comment:
I'm afraid you'll need to revert these changes. `Thread.Sleep(...)` is not available for **UAP10.0** TFM (see https://github.com/sshnet/SSH.NET/wiki/Conditional-compilation-symbols for more information).